app/views/layouts/_container_auth.html.haml

```diff
@@ -0,0 +1,24 @@
+- change_url ||= "form_field_changed"
+- legendtext ||= _("Credentials")
+
+%h3= legendtext
+#auth_tabs{:style => "display:none;"}
+  %ul.tab
+    %li
+      %a{:href => "#default"}= _("Default")
+
+  #default{:name => "Default"}
+    %table.style1
+      %tbody
+        = render(:partial => "/layouts/auth_bearer_token",
+                   :locals  => {:pfx          => "bearer",
+                                :change_url   => change_url,
+                                :ujs_button   => true,
+                                :record       => record})
+    %span{:style => "color:black"}= _("Optional.")
+
+%hr
+%script{:type => "text/javascript"}
+  -# method takes hash that can have 4 keys: tabs div id, active tab label,
+  -# url to go to when tab is changed, and whether to check for abandon changes or not
+  miq_jquery_tabs_init({tabs_div: "auth_tabs"})
```

Comment:
@oschreib have you verified this lately? I think `miq_jquery_tabs_init` got replaced by `miq_tabs_init`.

See #3396